typedb-src/modules/ROOT/partials/docker.adoc

```diff
@@ -28,16 +28,17 @@ docker pull vaticle/typedb:latest
 The TypeDB container exposes TypeDB's default port, `1729`, and uses its default data directory,
 `/opt/typedb-all-linux/server/data`.
 
-The following command starts a named TypeDB container, maps its exposed port to the host machine, and creates a named
-volume mapped to the data directory -- making it persistent across container restarts.
+The following command starts a named TypeDB container, maps its connection port to the host machine, and creates a
+docker volume named `try-typedb` mapped to the data directory -- making it persistent across container restarts.
 
 [,bash]
 ----
 docker run --name try-typedb -d -v try-typedb-data:/opt/typedb-all-linux/server/data/ -p 1729:1729 --platform linux/amd64 vaticle/typedb:latest
 ----
 
-The `--platform linux/amd64` parameter is required to run the TypeDB container on macOS with an `ARM64`/`AArch64`
-architecture.
+The `--platform linux/amd64` parameter is required to run the TypeDB container on macOS with the `ARM64`/`AArch64`
+architecture. The support for the `ARM64` architecture on Linux will be available in one of the future versions of
```

Comment:
It's shorter to say `... Support for `linux/arm64` will be released in a future version of TypeDB.`

This way you're using the docker terminology

Reply:
Yes, I was trying to avoid the Docker terminology because I was thinking it was a general case of `ARM64` on Linux support. Thanks.
```suggestion
architecture. Support for `linux/arm64` will be released in a future version of
```